server/index.js

```diff
@@ -91,6 +91,10 @@ app.patch('/api/update-project/:projectId', (req, res, next) => {
 app.delete('/api/delete-project/:projectId', (req, res, next) => {
   const projectId = req.params.projectId;
   const sql = `
+    with "deletePermissions" as (
+      delete from "projectPermissions"
+            where "projectId" = $1
+    )
     delete from "projects"
           where "projectId" = $1
       returning *
```

Comment:
You'll need to refactor your create/update/delete routes to use the `req.user.userId` in their `where` clauses so that only the owner of the project can do specific things. as they are now, your routes allow anybody to do anything to any project. You'll need an authorization middleware.